src/components/player/index.jsx

```
@@ -121,40 +132,49 @@ const Player = ({ video, isOpen, onClose }) => {
-          {totalComments > 0 && (
-            <button className="player-button">
-              {totalComments === 1
-                ? `${totalComments} comment`
-                : `${totalComments} comments`}
+        {!isTop && (
+          <div className="player-buttons-container">
+            <button
+              onClick={handleLike}
+              className="player-button"
+              disabled={isLoading}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)